assets/prometheus-k8s/prometheus.yaml

```diff
@@ -153,10 +153,12 @@ spec:
       requests:
         cpu: 1m
         memory: 25Mi
+    terminationMessagePolicy: FallbackToLogsOnError
     volumeMounts:
     - mountPath: /etc/tls/grpc
       name: secret-grpc-tls
   - name: prometheus
+    terminationMessagePolicy: FallbackToLogsOnError
```

Comment:
the operator should already set the field

Reply:
+ we already set it for prom containers https://github.com/openshift/cluster-monitoring-operator/blob/6b4252102a2aad5544be7f8b08419e73fd2621d9/assets/prometheus-k8s/prometheus.yaml#L104 maybe we should remove that as well (we already manually set it for other containers as well: https://github.com/search?q=repo%3Aopenshift%2Fcluster-monitoring-operator%20terminationMessagePolicy&type=code, maybe remove that and let the jsonnet loop inject it)

Reply:
(to test that this always stand + to test other properties we want to ensure for all "resource" on the 2 namespaces, origin monitor tests can be helpful https://github.com/openshift/origin/blob/master/pkg/monitortests/README.md: create a watch on the resource and run check on it at the end, we could create a ticket to add a check for terminationMessagePolicy and maybe others in the future)

Reply:
Yeah I missed to remove the individual sets. The idea here was to simply set it for all containers in a jsonnet util function instead of remembering to set it for all containers that need it.
I have removed the components level settings locally but something is not working right yet, so
/hold

But happy to discuss is this makes sense at all. I was just going off the old Jira linked to this. Good chance, this isn't the way.